tools/mason/MasonHelp.chpl

```diff
@@ -193,6 +193,7 @@ proc masonExternalHelp() {
   writeln("    uninstall                   Uninstall an external package");
   writeln("    info                        Show information about an external package");
   writeln("    find                        Find information about installed external packages");
+  writeln("    -V                          Display Spack version");
```

Comment:
@ben-albrecht I did try to add a `--version` before, but on running it I got an error as follows: 
```
<command-line arg>:2: error: Configuration variable 'version' is missing its initialization value 
```

Reply:
Oh! That's because you define `config const version` above. Let's just make that a `const version` instead. (`const spackVersion` with my name suggestion)